manticore/platforms/linux.py

```diff
@@ -264,12 +264,21 @@ class Linux(Platform):
     This class emulates the most common Linux system calls
     '''
 
-    def __init__(self, program, argv=None, envp=None):
-        '''
+    def __init__(self, program, argv=None, envp=None, **options):
```

Comment:
I feel like using **options at this point isn't changing the API any less, just in a less obvious way. By changing the model (platform, technically) API, do you mean altering this declaration line?